tools/wheelmaker.py

```diff
@@ -398,6 +403,10 @@ def main() -> None:
                       encoding="utf-8") as metadata_file:
                 metadata = metadata_file.read()
 
+        if arguments.requires_file:
+            with open(arguments.requires_file) as fp:
+                additive_requires_list = ["Requires-Dist: {}".format(line) for line in fp.read().strip().split("\n")]
+                metadata += "\n" + "\n".join(additive_requires_list)
```

Comment:
Won't that leave you with two \n in a row, so everything below will be interpreted as description?

Can we have a test?